src/components/ButtonPanel.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const ButtonPanel = () => (
+  <>
+    <div>
+      <div>AC</div>
+      <div>+/-</div>
+      <div>%</div>
+      <div>/</div>
+    </div>
+    <div>
+      <div>7</div>
+      <div>8</div>
+      <div>9</div>
+      <div>X</div>
+    </div>
+    <div>
+      <div>4</div>
+      <div>5</div>
+      <div>6</div>
+      <div>-</div>
+    </div>
+    <div>
+      <div>1</div>
+      <div>2</div>
+      <div>3</div>
+      <div>+</div>
+    </div>
+    <div>
+      <div>0</div>
+      <div>.</div>
+      <div>=</div>
+    </div>
+  </>
+);
```

Comment:
### [Optional]
Your code appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code, In other words; you repeated the `Button` component multiple times.

You can use [objects.key](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)  and `map` to loop through the objects. `objects.keys` for the outer loop and `map` for the inner loop.

Example; 
```
const GroupButtons = {
  Group 1: ['AC', '+/-', '%', '÷'],
  Group 2: ['7', '8', '9', 'X'],
   ............
}
```
OR

[Creating a two-dimensional array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#creating_a_two-dimensional_array) and implement nested loop with a map.
EXample;
```
  const buttons = [
    ['AC', '+/-', '%', '÷'],
    ['7', '8', '9', 'X'],
    ..........
  ];
```

Reply:
Nice! I used Emmet to type for me lol, but I will surely implement your suggestion.
Thanks a lot! :+1: